### ISSUE TITLE
Add a Separate option to make self-closing tag's closing-bracket newline

### ISSUE THREAD

**Issue by pycodev** [vuejs/eslint-plugin-vue#990]
**Please describe what the rule should do:**
hope to Add a Separate option on the rule, `vue/html-closing-bracket-newline` , to make self-closing tag's closing-bracket newline.

**What category should the rule belong to?**

- [x ] Enforces code style
- [ ] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
so that we can respectively control the un-self-closing tag and self-closing tag's closing-bracket style. like below

```vue
<!-- 
the rule may like this: 
{
  "vue/html-closing-bracket-newline": ["error", {
    "singleline": "never",
    "multiline": "never",
    "selfclose": "always"
  }]
}
 -->

<component attr1="value1" attr2="value2" />

<component
    attr1="value1"
    attr2="value2">
    content
</component>

<component
    attr1="value1"
    attr2="value2"
/>

```

**Comment by imlinus** [comment]
I'm also looking for something similar 👍
